src/encoded/ingestion/vcf_utils.py

```diff
@@ -335,7 +338,7 @@ def cast_field_value(self, t, value, sub_type=None):
         elif t == 'array':
             if sub_type:
                 if not isinstance(value, list):
-                    items = self.fix_encoding(value).split('&') if sub_type == 'string' else value
+                    items = self.fix_encoding(value).split('&')
                 else:
                     items = value
```

Comment:
The elimination of the useless branch makes sense, though I might prefer to see the clauses spelled out more clearly. In the case of a number, do you intend to get an error or use the value? That is (untested):
```suggestion
                if isinstance(value, str):
                    items = self.fix_encoding(value).split('&')
                elif isinstance(value, list):
                    items = value
                else:
                    raise ValueError("Unexpected data where a string or a list was expected: %r" % value)
```
Or you may want to do (again untested):
```suggestion
                if isinstance(value, str):
                    items = self.fix_encoding(value).split('&')
                elif isinstance(value, list):
                    items = value
                else:  # perhaps a number? can this even happen?
                   items = [str(value)]
```